core-bundle/config/services.yaml

```diff
@@ -443,6 +443,11 @@ services:
     contao.insert_tag.php_function_flag:
         class: Contao\CoreBundle\InsertTag\Flag\PhpFunctionFlag
 
+    contao.insert_tag.resolver.asset:
```

Comment:
The event insert tag uses `contao_calendar.insert_tag.event` without the `\Resolver\` sub-namespace. What is our logic behind when to use the sub-namespace and when not?

Reply:
imho it is different for the Core Bundle as it acts as a library here since it has the base classes and interfaces etc. there - thus it is sensible to have sub namespaces.

Within the `App\` or another bundle I personally would only use the `InsertTag` namespace - unless I have a lot of insert tag resolvers and insert tag flags (which is unlikely).

Reply:
Yes, the subnamespace is only in the core-bundle because it has many other classes in the `InsertTag` namespace